pkg/assets/configs.go

```diff
@@ -0,0 +1,52 @@
+package assets
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/golang/glog"
+)
+
+var (
+	configAssets = []string{
+		"kube-apiserver-config-aggregator-client-ca.yaml",
+		"kube-apiserver-config-client-ca.yaml",
+		"kube-apiserver-config-etcd-serving-ca.yaml",
+		"kube-apiserver-config-kubelet-serving-ca.yaml",
+		"kube-apiserver-config-sa-token-signing-certs.yaml",
+	}
+)
+
+func LoadLocalConfigs(configDir string) KubeAPIServerConfigMapsConfig {
```

Comment:
would change config to configmap in this context. Otherwise, we have too many "configs" around.